lib/mockito.dart

```diff
@@ -404,3 +531,14 @@ void logInvocations(List<Mock> mocks) {
     print(inv.toString());
   });
 }
+
+/// Should only be used during Mockito testing.
```

Comment:
Well... I don't know. I don't see any harm in making in public. It might be a feature for power users... or if someone writes improper mocks like my tests do... for some reason. Maybe other libraries that build on mockito's functionality.